source/tools/testConsistency/ClientApp/views/index.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+    <h1>Consistent Cards</h1>
+    <form action = "/" method=POST>
+      <textarea placeholder="card data" name ="cardData"></textarea><br>
+      <textarea placeholder="card template" name ="cardTemplate"></textarea><br>
```

Comment:
totally fine for now, but if you want to snazz this up later, let me know -- we can get this hooked up with a proper editor (like the one used in the designer)